membership/README.md

```diff
@@ -0,0 +1,141 @@
+| more than $4 billion    | 16,200          | 6,750                         |
+
+**Bedework supporting subscription:** $500
+
+**CAS supporting subscription:** $500
+
+| uPortal Supporting Subscription level | Dues in US dollars |
+| ------------------------------------- | ------------------ |
+| Bronze - uPortal Friend               |  1,000             |
+| Silver - uPortal Booster              |  5,000             |
+| Gold - uPortal Sustainer              | 10,000             |
+| Platinum - uPortal Champion           | 20,000             |
+
+### Canada (CAD)
+
+| Annual Budget (CAD) | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |  2,851          | 1,188                         |
+| less than $100 million  |  6,177          | 2,574                         |
+| less than $750 million  | 12,830          | 5,346                         |
+| less than $4 billion    | 16,632          | 6,930                         |
+| more than $4 billion    | 21,384          | 8,910                         |
+
+**Bedework supporting subscription:** $660
+
+**CAS supporting subscription:** $660
+
+| uPortal Supporting Subscription level | Dues in Canadian dollars |
+| ------------------------------------- | ------------------------ |
+| Bronze - uPortal Friend               |  1,320                   |
+| Silver - uPortal Booster              |  6,600                   |
+| Gold - uPortal Sustainer              | 13,200                   |
+| Platinum - uPortal Champion           | 26,400                   |
+
+### Eurozone (Euro)
+
+| Annual Budget (Euro)    | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |  2,052          |   855                         |
+| less than $100 million  |  4,446          | 1,852.50                      |
+| less than $750 million  |  9,234          | 3,847.50                      |
+| less than $4 billion    | 11,970          | 4,987                         |
+| more than $4 billion    | 15,390          | 6,412.50                      |
+
+**Bedework supporting subscription:** 475 Euros
+
+**CAS supporting subscription:** 475 Euros
+
+| uPortal Supporting Subscription level | Dues in Euros |
+| ------------------------------------- | ------------------------ |
+| Bronze - uPortal Friend               |    950                   |
+| Silver - uPortal Booster              |  4,750                   |
+| Gold - uPortal Sustainer              |  9,500                   |
+| Platinum - uPortal Champion           | 19,000                   |
+
+### United Kingdom (UK Pounds Sterling)
+
+| Annual Budget (UKP)     | Core Foundation | Sakai Supporting Subscription |
+| ----------------------- | --------------- | ----------------------------- |
+| less than $25 million   |    729          |   729                         |
```

Comment:
It's weird that in the UK Core Foundation dues and Sakai Supporting Subcription dues are equal whereas in other localities they're not. So weird that maybe this is an error?